hebrewreaderserver.py

```diff
@@ -73,15 +73,15 @@ def do_generate_reader(self, fmt=['pdf'],
         pdf = tempfile.mkstemp(suffix='.pdf', prefix='reader')[1]
 
         try:
-            with time_limit(10):
+            with time_limit(30):
                 generate(passages,
                         include_voca is not None and include_voca,
                         combine_voca is not None and combine_voca,
                         clearpage_before_voca is not None and clearpage_before_voca,
                         text_size is not None and int(text_size[0]) > 0,
                         text_size is not None and int(text_size[0]) > 1,
                         tex, None if fmt == 'tex' else pdf,
-                        TEMPLATES, quiet=True)
+                        TEMPLATES, lang, quiet=False)
```

Comment:
Please use `lang[0]` here (it is an artifact of the server interface that we get a list with one element, and `generate` should be agnostic to the interface).